libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts

```diff
@@ -44,18 +38,6 @@ describe("recordedDataView", () => {
 });
 
 describe("execution factory", () => {
-    it("should load result with uriRefs by default", async () => {
-        const result = await recordedBackend(ReferenceRecordings.Recordings)
-            .workspace("reference-workspace")
-            .execution()
-            .forDefinition(
-                ReferenceRecordings.Scenarios.BarChart.SingleMeasureWithViewBy.execution.definition,
-            )
-            .execute();
-
-        assertExpectedRefs(result, isUriRef);
```

Comment:
Should we change the default to idRefs, and just update the tests (assert isIdRef)?

Reply:
Same as before test it uri by default thats why test deleted

Reply:
`isIdRef` does not do the trick?